README.md

```diff
@@ -153,29 +153,18 @@ Use the [`toledo`](https://github.com/sigp/lighthouse/pull/1874) branch (until t
 
 ### Nimbus
 
-TODO: which git branch to run?
+Use the [`devel`](https://github.com/status-im/nimbus-eth2/tree/devel) branch.
 
-#### Beacon node
+#### Stand-alone beacon node
 
-Prepare a binary with insecure/experimental features enabled:
 ```shell script
-make -j8 LOG_LEVEL="TRACE" NIMFLAGS="-d:insecure" beacon_node
-```
-
-Requires a special json config file.
-```
-  --network="nimbus_config.json"
+make toledo
 ```
 
-Config file looks like [`nimbus_config.json`](./nimbus_config.json)
+#### Beacon node with a validator client attached
```

Comment:
Why not stand-alone VC? Is that broken? I would prefer to be on the edge, report bugs, and get a stand-alone client ready

Reply:
Perhaps my wording is confusing. I meant that this would be running two processes - one for the beacon node and one for the validator client. "Attached to" was merely referring to the RPC connection between them.